packages/cvs/build.sh

```diff
@@ -13,6 +13,7 @@ cvs_cv_func_printf_ptr=yes
 ac_cv_header_syslog_h=no
 --disable-server
 --with-external-zlib
+--with-editor=$TERMUX_PREFIX/bin/vi
```

Comment:
If this ends up as being the default editor used by CVS, this should probably be `nano`,
as we don't ship `vi` in the bootstrap.
```suggestion
--with-editor=$TERMUX_PREFIX/bin/nano
```

Reply:
Thank you that is my mistake. I looked for default editors in an installation of Termux that I thought was clean but it was not. You are correct, in the default clean installed app there is only `nano`.

Reply:
Probably it will be better to use `$TERMUX_PREFIX/bin/editor` because it is a default symlink to editor used for system. But it may not appear on build, only on device.

Reply:
@twaik that is also a good idea, though I just noticed, that when I test the commands `nano` and `editor`  in the clean installed Termux app, at least for me there seem to be different default keybinds in the version of `nano` that appears when invoked from the `editor` command, while when I invoke `nano` directly the keybinds seem more familiar to someone who used `nano` on some other distros. 

For example, with the command `nano`, the keybind to Save seems to be Ctrl + O
On the other hand, when I use the command `editor`, the keybind to Save seems to be Ctrl + W instead

Do you know if this behavior is intended, and if so then, which version of launching `nano` do you think is a better "default" experience?

I am testing using this app clean installed https://f-droid.org/repo/com.termux_1020.apk

Reply:
> For example, with the command `nano`, the keybind to Save seems to be Ctrl + O
On the other hand, when I use the command `editor`, the keybind to Save seems to be Ctrl + W instead

I think you should open issue about that, seems to be separate case we should investigate.

Reply:
Ok actually it is only reproducible in the preinstalled bootstrap and not in the current packages.
Therefore the issue with `editor` can be solved  by any of `pkg upgrade`, `pkg install vim` or `pkg install emacs` with an internet connection.